android/src/main/java/com/reactnativekeyboardcontroller/views/EdgeToEdgeReactViewGroup.kt

```diff
@@ -200,4 +202,20 @@ class EdgeToEdgeReactViewGroup(private val reactContext: ThemedReactContext) : R
     }
   }
   // endregion
+
+  // region external methods
+  fun forceStatusBarTranslucent(isStatusBarTranslucent: Boolean) {
+    UiThreadUtil.runOnUiThread {
```

Comment:
Maybe we should run it from UI thread from module package?
Component shouldn't know anything abouth the thread in this situation 🤔